lib/std/math/log_int.zig

```diff
@@ -0,0 +1,79 @@
+const std = @import("../std.zig");
+const math = std.math;
+const testing = std.testing;
+const assert = std.debug.assert;
+const Log2Int = math.Log2Int;
+
+/// Returns the logarithm of `x` for the provided `base`, rounding down to the nearest integer.
```

Comment:
Is -1.5 also rounded down or do you mean rounding to 0?

Reply:
There are no negative logarithms in this case, since both `base` and `x` are strictly positive.